### DISCUSSION TITLE
CRITICAL: Trakt authentication error: Unauthorized - OAuth must be provided

### DISCUSSION THREAD

**Discussion by kimpenhaus** [Taxel/PlexTraktSync#392] [Q&A] [answered]
Hi all,

I am experiencing this since some time now and wonder if I am the only one and what might be the cause.

I am running current main in an pipenv on console. Steps I did:

1) `git pull` and run `pipenv install` for latest changes:

<img width="607" alt="image" src="https://user-images.githubusercontent.com/575109/126834878-892f371b-bba9-4be1-996a-1cedca33cc2c.png">

2) `rm .env` and `rm .pytrakt.json` for clean start

3) log into plex and log into trakt - as far as I can tell it's successful:

<img width="1106" alt="image" src="https://user-images.githubusercontent.com/575109/126835211-2354550e-d569-4395-a7f7-6802cba2fd73.png">

<img width="1424" alt="image" src="https://user-images.githubusercontent.com/575109/126835364-08cd6abb-c996-4273-bd90-b5431a8f2366.png">

4) try to run sync `pipenv run python main.py`

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/575109/126835735-2eec2534-f815-41cb-b15d-7bcd225c2c47.png">

I tried several other sync commands like `./plex_trakt_sync.sh sync` or `pipenv run python main.py sync`... obviously resulting in the same errors (as they - at least from what I understood - all do the same)

any help is appreciated - thanks 👋🏼  

Forgot to paste the output of the `pipenv graph`

<img width="609" alt="image" src="https://user-images.githubusercontent.com/575109/126836525-838d3ced-2f75-4478-816e-f44cdd3743d6.png">

**Comment by chrillep** [comment]
have you tried `pipenv --clear` ?

```
 --clear                         Clears caches (pipenv, pip, and pip-tools).
                                  [env var: PIPENV_CLEAR]
```

**Reply by kimpenhaus** [reply]
@chrillep yeah tried several things - also deleted and pulled new from github and ran from scratch... but none of it worked. funny thing is that it was working in the past and stopped by a specific commit (but I can't tell which).

but I switched to the docker container now and that is working as expected - so I am currently not focusing on this problem I had.

**Comment by glensc** [comment]
Bugs must be reported as Issues. But since this is just a question, I'll say "Doesn't happen for me".

**Comment by kimpenhaus** [comment]
ja I wouldn't call this a bug - might be a problem with my specific machine, that's why I asked.

**Reply by chrillep** [reply]
i got this issue once, i dont recall exactly how i resolved it. But i believe i just removed the whole repo folder and cloned it again. :)

**Reply by glensc** [reply]
It might be a bug as well. Discussions don't have such visibility as issues to developers, can't see a number of open questions for example.

**Reply by glensc** [reply, marked as answer]
This must have been due `~/.pytrakt.json` file presence interfering:
- https://github.com/Taxel/PlexTraktSync/issues/20#issuecomment-954825293

Fix is to remove `~/.pytrakt.json` file.
